### ISSUE TITLE
[wishlist) polyline direction

### ISSUE THREAD

**Issue by cascafico** [umap-project/umap#273]
Hi all!

Is there a property (like dash array) that shows direction (ie: direcrtional symbols) along a polyline?
If not, it would be nice-to-have for cases such as race routes.

**Comment by yohanboniface** [comment]
Yes, that's somewhere on the roadmap, see: https://github.com/umap-project/Leaflet.Storage/issues/103 :)

**Comment by cascafico** [comment]
Merci :-)
looking forward for that

**Comment by robinmetral** [comment]
Can't see this on the [new roadmap](https://github.com/orgs/umap-project/projects/1/views/1). Is it still a plan? :slightly_smiling_face:

**Comment by yohanboniface** [comment]
Would implementing this answer the need ? https://github.com/umap-project/umap/issues/318

**Comment by robinmetral** [comment]
For my use case, I think it would, yes :slightly_smiling_face: Can't speak for @cascafico though

**Comment by yohanboniface** [comment]
Path decoration has been implemented
